packages/nestjs/src/decorators/sentry-traced.ts

```diff
@@ -20,6 +20,10 @@ export function SentryTraced(op: string = 'function') {
         },
       );
     };
+
+    // preserve the original name on the decorated function
+    Object.defineProperty(descriptor.value, 'name', { value: originalMethod.name, configurable: true });
```

Comment:
I think we need to set this to be `writeable: true` as well, and maybe also `enumerable: true` if `name` is an `enumerable` property by default.